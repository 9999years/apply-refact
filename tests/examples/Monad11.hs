yes = do x <- return $ y + z; foo x