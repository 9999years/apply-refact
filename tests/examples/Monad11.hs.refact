[("tests/examples/Monad11.hs:1:10-28: Suggestion: Use let\nFound:\n  x <- return $ y + z\nPerhaps:\n  let x = y + z\n",[Replace {rtype = Stmt, pos = SrcSpan {startLine = 1, startCol = 10, endLine = 1, endCol = 29}, subts = [("lhs",SrcSpan {startLine = 1, startCol = 10, endLine = 1, endCol = 11}),("rhs",SrcSpan {startLine = 1, startCol = 24, endLine = 1, endCol = 29})], orig = "let lhs = rhs"}])]
