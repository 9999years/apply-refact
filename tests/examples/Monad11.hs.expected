yes = do let; x = y + z foo x
